testonly/hammer/maphammer/main.go

```diff
@@ -53,8 +53,10 @@ var (
 	outLog          = flag.String("log_to", "", "File to record operations in")
 	seed            = flag.Int64("seed", -1, "Seed for random number generation")
 	operations      = flag.Uint64("operations", ^uint64(0), "Number of operations to perform")
-	minLeaves       = flag.Int("min_leaves", 0, "Minimum count of leaves to affect per-operation")
-	maxLeaves       = flag.Int("max_leaves", 10, "Maximum count of leaves to affect per-operation")
+	minLeaves       = flag.Int("min_leaves", 0, "Minimum count of leaves to read per-operation")
```

Comment:
Is there any reason to reuse the old names? Maybe name them explicitly as `min/max_leaves_read`?